agent/containers/images/repo.yml.j2

```diff
@@ -1,6 +1,6 @@
 ---
 repos:
-  - tag: pbench
+  - tag: pbench{{ test_suffix }}
```

Comment:
Where does this fit in? Is there some role that got modified and should be part of this PR?

Reply:
See line 215 of the `Makefile`.